### ISSUE TITLE
Add logging

### ISSUE THREAD

**Issue by SaadBazaz** [blomqma/next-rest-framework#80]
[ Pino](https://github.com/pinojs/pino-nextjs-example) could do a good job.

**Comment by blomqma** [comment]
Yeah I think the existing logging could be improved for sure. What kind of information would you expect to see logged by the framework?

**Comment by SaadBazaz** [comment]
Something more on the lines of how NestJS manages em. For production, they throw JSON in the terminal. For dev, it's formatted.

Mostly we see:
Status code
Route
Time hit
User Agent
Response

With the right colors.
